doc/ref_impl/cntt-ri2/chapters/chapter02.md

```diff
@@ -22,10 +22,20 @@ interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
 ## 2.2 Reference Architecture Specification
 
-| RA2 Section | RA2 Reference  | Description | Requirement for Basic Profile | Requirement for Network Intensive Profile| RI2 Traceability |
-|---|---|---|---|---|---|
-|Kubernetes Nodes|`ra2.ch.001`|Huge Pages|Not required|It must be possible to enable Huge Pages (2048KiB and 1048576KiB) within the Kubernetes Node OS, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|_TBC_|
-|||||||
+| RA2 Section | RA2 Reference  | Specification | Requirement | Requirement for Basic Profile | Requirement for Network Intensive Profile | RI2 Traceability |
+|---|---|---|---|---|---|---|
+|Kubernetes Nodes|`ra2.ch.001`|Huge Pages|It must be possible to enable Huge Pages (2048KiB and 1048576KiB) within the Kubernetes Node OS, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|Must support|Not required|[4.3.1](./chapter04.md#431-installation-on-bare-metal-infratructure)|
+|Kubernetes Nodes|`ra2.ch.002`|SR-IOV Capable NICs|When hosting workloads matching the Network Intensive profile, the physical machines on which the Kubernetes Nodes run must be equipped with NICs that are SR-IOV capable.|Not required|Must support|[3.3](./chapter03.md#33-infrastructure-requirements)|
+|Kubernetes Nodes|`ra2.ch.003`|SR-IOV Virtual Functions|When hosting workloads matching the Network Intensive profile, SR-IOV virtual functions (VFs) must be configured within the Kubernetes Node OS, as the SR-IOV Device Plugin does not manage the creation of these VFs.|Not required|Must support|[4.3.1](./chapter04.md#431-installation-on-bare-metal-infratructure)|
```

Comment:
Should it only be that the creation of the VFs in "within the Kubernetes Node OS" as the configuration can be managed by a SR-IOV CNI plugin?
```suggestion
|Kubernetes Nodes|`ra2.ch.003`|SR-IOV Virtual Functions|When hosting workloads matching the Network Intensive profile, SR-IOV virtual functions (VFs) must be created within the Kubernetes Node OS, as the SR-IOV Device Plugin does not manage the creation of these VFs.|Not required|Must support|[4.3.1](./chapter04.md#431-installation-on-bare-metal-infratructure)|
```

Reply:
Yeah, I think "created" is a better choice of word here